src/main/java/com/walmartlabs/x12/standard/X12Group.java

```diff
@@ -13,9 +13,16 @@
 See the License for the specific language governing permissions and
 limitations under the License.
  */
-package com.walmartlabs.x12.common;
+package com.walmartlabs.x12.standard;
 
-public class GroupHeader {
+import com.walmartlabs.x12.X12TransactionSet;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class X12Group {
+
+    // Header Data Elements
```

Comment:
I wanted to know the reason for why are we not creating separate class for GsHeader(GS-group header) and GeTrailer(G@-group trailer)

Reply:
Probably better to combine ISA and ISE